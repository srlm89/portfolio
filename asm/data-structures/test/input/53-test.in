ERASE actual/53-test.out
INSERT 28 8.926662 The Last Samurai [2004] [720p]
PRINT_IN_ORDER actual/53-test.out
INSERT 0 0.237190 Drag Me To Hell [720p]
INSERT 14 5.040995 Godzilla 720p
REMOVE_ID 14
PRINT_IN_ORDER actual/53-test.out
INSERT 36 4.588228 Shutter   720p
INSERT 7 1.823887 Monster Inc [2001] [720p]
INSERT 12 2.254774 Pursuit_Of_Happyness[2006]   720p
REMOVE_ID 28
PRINT_IN_ORDER actual/53-test.out
INSERT 39 8.842121 Jumanji [720p]
INSERT 57 0.361246 Toy.Story.1995.720p.HDTV.x264-CtrlHD
INSERT 10 6.350035 Night Train   720p
INSERT 31 9.776077 A Bug's Life [720p]
REMOVE_ID 12
REMOVE_ID 7
PRINT_IN_ORDER actual/53-test.out
INSERT 23 5.205005 Fantastic Four, Rise Of The Silver Surfer    1080p
INSERT 41 1.150291 Ongbak 2003 720p
INSERT 42 8.191014 Terminator Salvation Director's cut  ( 2009)    1080p
INSERT 52 0.478930 Firewall 720p
INSERT 15 7.905991 Pitch Black [720p]
REMOVE_ID 10
REMOVE_ID 15
PRINT_IN_ORDER actual/53-test.out
INSERT 17 0.660664 The Day After Tomorrow 1080p
INSERT 21 2.620742 Rush Hour 1-3   720p
INSERT 37 8.631833 Corpse Bride   720p
INSERT 22 8.984420 Batman Dark Knight   1080p
INSERT 67 2.287852 Toy.Story.1995.720p.HDTV.x264-CtrlHD
INSERT 6 3.902630 James Bond 16 - Licence to Kill [1989]
REMOVE_ID 57
REMOVE_ID 41
REMOVE_ID 21
PRINT_IN_ORDER actual/53-test.out
INSERT 63 6.882587 Cloverfield   1080p
INSERT 72 2.201533 Harry Potter Collection    720p
INSERT 75 3.758971 Apollo 13   1080p
INSERT 54 5.236005 Ratatouille [2007]
INSERT 2 0.854142 Chicken Run [2000] [720p]
INSERT 64 3.457095 Shinjuku Incident
INSERT 3 2.345400 Ice Age 1 [2002] [720p]
REMOVE_ID 54
REMOVE_ID 75
REMOVE_ID 31
PRINT_IN_ORDER actual/53-test.out
INSERT 60 4.309528 G Force    720p
INSERT 9 2.473298 James Bond 11 - MoonRaker [1979]
INSERT 1 9.841776 Bad Boy I [1995] [720p]
INSERT 26 0.049906 enemy at the gates    1080p
INSERT 70 7.754090 Seven Pounds    720p
INSERT 30 3.564558 Erin Brockovich 2000.720p
INSERT 86 6.852589 Death Sentence   1080p
INSERT 74 2.967514 Drag Me To Hell [720p]
REMOVE_ID 67
REMOVE_ID 63
REMOVE_ID 6
REMOVE_ID 74
PRINT_IN_ORDER actual/53-test.out
INSERT 40 4.108923 The Mummy 1-2   1080p
INSERT 62 7.572142 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p]
INSERT 77 5.104489 Lord Of The Ring 3 - The Return of the King (ExtEd) [1080p]
INSERT 91 1.054237 Top.Gun.1986.720p.HDDVD.x264-SEPTiC
INSERT 32 1.315114 Counterfeiters.LIMITED.720p.Bluray.x264-SEPTiC
INSERT 84 6.433499 James Bond 11 - MoonRaker [1979]
INSERT 18 5.708070 A.Beautiful.Mind.2001.720p.HDTV.x264-S26
INSERT 47 5.977164 Kung Fu Panda [2008]   720p
INSERT 93 6.090223 Bad Boy I [1995] [720p]
REMOVE_ID 30
REMOVE_ID 60
REMOVE_ID 0
REMOVE_ID 62
PRINT_IN_ORDER actual/53-test.out
INSERT 45 9.855380 Black.Mask.Blu-ray.720p.x264.DTS-HDC.AfterFX@SiLUHD
INSERT 38 0.922589 Juno 2007 720p
INSERT 71 4.864067 Deep Sea
INSERT 81 9.862417 Equilibrium 720p
INSERT 79 7.089044 National Tressure 1 - 1080p
INSERT 29 4.144655 Face.Off.1997.720p
INSERT 27 0.407093 Assassination of a High School President 720p
INSERT 104 6.657750 Ghost Rider [2007] [720p]
INSERT 101 2.422137 Dragon Wars   720p
INSERT 33 1.901524 Casablana.1942.1080p.BluRay.x264.Codres
REMOVE_ID 22
REMOVE_ID 84
REMOVE_ID 86
REMOVE_ID 32
REMOVE_ID 104
PRINT_IN_ORDER actual/53-test.out
INSERT 19 6.684097 James Bond 03 - GoldFinger [1964]
INSERT 89 1.512438 Night Train   720p
INSERT 83 7.945302 8 Mile [2002] [720p]
INSERT 82 2.474496 Transformer 1 & 2 Collection   1080p
INSERT 51 4.375161 The Dark Knight [2008]   1080p
INSERT 11 3.092086 American Virgin 720P
INSERT 8 8.851587 Pirates of Caribbean 1-2-3 Collection   1080p
INSERT 35 0.955647 Eagle Eye    1080p
INSERT 88 9.753327 Fifty Dead Men Walking   1080p
INSERT 76 9.008009 Eurotrip    720p
INSERT 102 1.526266 10,000 BC 720p
REMOVE_ID 39
REMOVE_ID 40
REMOVE_ID 77
REMOVE_ID 29
REMOVE_ID 37
PRINT_IN_ORDER actual/53-test.out
INSERT 61 8.070845 Pursuit_Of_Happyness[2006]   720p
INSERT 90 6.126213 X-Men 1 - 1080p
INSERT 16 3.309936 Race To Witch Mountain [1080p]
INSERT 34 0.740428 SAW IV
INSERT 106 4.813766 Dragon Wars   720p
INSERT 53 5.482094 Chicken Little 1080p
INSERT 119 9.744142 Death Sentence   1080p
INSERT 108 1.580269 Dragonball Evolution [2009] [720p]
INSERT 69 3.468441 Yes Man   720p
INSERT 5 1.150963 The Matrix   1080p
INSERT 68 4.854085 Indiana.Jones.and.the.Kingdom.of.the.Crystal.Skull    1080p
INSERT 123 7.144789 Species - The Awakening   720p
REMOVE_ID 5
REMOVE_ID 88
REMOVE_ID 35
REMOVE_ID 9
REMOVE_ID 1
REMOVE_ID 11
PRINT_IN_ORDER actual/53-test.out
INSERT 117 7.868404 National Tressure 1 - 1080p
INSERT 98 5.818378 Monster Inc [2001] [720p]
INSERT 50 6.134728 Lord Of The Ring  - Return Of The King   720p
INSERT 96 0.660276 Sky.Captain.And.The.World.Of.Tomorrow[www.x1949x.com]    720p
INSERT 20 3.718962 Blood.Diamond.2006.1080p.HDDVD.x264-DEFiNiTE
INSERT 4 6.145100 contents.txt
INSERT 25 0.976694 Blood River 720P
INSERT 55 5.938738 X-Men 2   1080p
INSERT 13 0.273591 Alone in the dark   720p
INSERT 135 6.449867 Death Sentence   1080p
INSERT 49 1.560757 Pitch Black [720p]
INSERT 140 2.605059 Golden Compass   720p
INSERT 133 2.038412 Finding Nemo [2003] [720p]
REMOVE_ID 70
REMOVE_ID 13
REMOVE_ID 68
REMOVE_ID 33
REMOVE_ID 64
REMOVE_ID 117
PRINT_IN_ORDER actual/53-test.out
INSERT 143 1.340041 Final Fantasy VII Advent Children [2005] [1080p]
INSERT 95 2.638061 Golden Compass   720p
INSERT 145 4.560221 CJ7   720p
INSERT 146 5.709145 Night At The Museum 2 [720p]
INSERT 105 9.477671 Around The World In 80 Days [2004] [720p]
INSERT 80 7.544803 Kung Fu Hustle   720p
INSERT 78 9.835793 Jurassic Park 1-3  1080p
INSERT 48 4.432253 American Beauty   720p
INSERT 97 4.410551 Shoot.Em.Up.720p.Bluray.x264-SEPTiC
INSERT 124 8.297857 Lord of war   720p
INSERT 125 9.944070 The Extreme Movie 720P
INSERT 65 8.500774 Knowing 2009 720p
INSERT 138 0.487034 Spiderman 1   1080p
INSERT 46 2.768796 High school musical II 2007    720p
REMOVE_ID 80
REMOVE_ID 50
REMOVE_ID 46
REMOVE_ID 27
REMOVE_ID 61
REMOVE_ID 23
REMOVE_ID 65
PRINT_IN_ORDER actual/53-test.out
