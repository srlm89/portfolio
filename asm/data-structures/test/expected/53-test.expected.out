[ 28 8.926662 The Last Samurai [2004] [720p] ]
[ 0 0.237190 Drag Me To Hell [720p] ][ 28 8.926662 The Last Samurai [2004] [720p] ]
[ 0 0.237190 Drag Me To Hell [720p] ][ 7 1.823887 Monster Inc [2001] [720p] ][ 12 2.254774 Pursuit_Of_Happyness[2006]   720p ][ 36 4.588228 Shutter   720p ]
[ 0 0.237190 Drag Me To Hell [720p] ][ 10 6.350035 Night Train   720p ][ 31 9.776077 A Bug's Life [720p] ][ 36 4.588228 Shutter   720p ][ 39 8.842121 Jumanji [720p] ][ 57 0.361246 Toy.Story.1995.720p.HDTV.x264-CtrlHD ]
[ 0 0.237190 Drag Me To Hell [720p] ][ 23 5.205005 Fantastic Four, Rise Of The Silver Surfer    1080p ][ 31 9.776077 A Bug's Life [720p] ][ 36 4.588228 Shutter   720p ][ 39 8.842121 Jumanji [720p] ][ 41 1.150291 Ongbak 2003 720p ][ 42 8.191014 Terminator Salvation Director's cut  ( 2009)    1080p ][ 52 0.478930 Firewall 720p ][ 57 0.361246 Toy.Story.1995.720p.HDTV.x264-CtrlHD ]
[ 0 0.237190 Drag Me To Hell [720p] ][ 6 3.902630 James Bond 16 - Licence to Kill [1989] ][ 17 0.660664 The Day After Tomorrow 1080p ][ 22 8.984420 Batman Dark Knight   1080p ][ 23 5.205005 Fantastic Four, Rise Of The Silver Surfer    1080p ][ 31 9.776077 A Bug's Life [720p] ][ 36 4.588228 Shutter   720p ][ 37 8.631833 Corpse Bride   720p ][ 39 8.842121 Jumanji [720p] ][ 42 8.191014 Terminator Salvation Director's cut  ( 2009)    1080p ][ 52 0.478930 Firewall 720p ][ 67 2.287852 Toy.Story.1995.720p.HDTV.x264-CtrlHD ]
[ 0 0.237190 Drag Me To Hell [720p] ][ 2 0.854142 Chicken Run [2000] [720p] ][ 3 2.345400 Ice Age 1 [2002] [720p] ][ 6 3.902630 James Bond 16 - Licence to Kill [1989] ][ 17 0.660664 The Day After Tomorrow 1080p ][ 22 8.984420 Batman Dark Knight   1080p ][ 23 5.205005 Fantastic Four, Rise Of The Silver Surfer    1080p ][ 36 4.588228 Shutter   720p ][ 37 8.631833 Corpse Bride   720p ][ 39 8.842121 Jumanji [720p] ][ 42 8.191014 Terminator Salvation Director's cut  ( 2009)    1080p ][ 52 0.478930 Firewall 720p ][ 63 6.882587 Cloverfield   1080p ][ 64 3.457095 Shinjuku Incident ][ 67 2.287852 Toy.Story.1995.720p.HDTV.x264-CtrlHD ][ 72 2.201533 Harry Potter Collection    720p ]
[ 0 0.237190 Drag Me To Hell [720p] ][ 1 9.841776 Bad Boy I [1995] [720p] ][ 2 0.854142 Chicken Run [2000] [720p] ][ 3 2.345400 Ice Age 1 [2002] [720p] ][ 9 2.473298 James Bond 11 - MoonRaker [1979] ][ 17 0.660664 The Day After Tomorrow 1080p ][ 22 8.984420 Batman Dark Knight   1080p ][ 23 5.205005 Fantastic Four, Rise Of The Silver Surfer    1080p ][ 26 0.049906 enemy at the gates    1080p ][ 30 3.564558 Erin Brockovich 2000.720p ][ 36 4.588228 Shutter   720p ][ 37 8.631833 Corpse Bride   720p ][ 39 8.842121 Jumanji [720p] ][ 42 8.191014 Terminator Salvation Director's cut  ( 2009)    1080p ][ 52 0.478930 Firewall 720p ][ 60 4.309528 G Force    720p ][ 64 3.457095 Shinjuku Incident ][ 70 7.754090 Seven Pounds    720p ][ 72 2.201533 Harry Potter Collection    720p ][ 86 6.852589 Death Sentence   1080p ]
[ 1 9.841776 Bad Boy I [1995] [720p] ][ 2 0.854142 Chicken Run [2000] [720p] ][ 3 2.345400 Ice Age 1 [2002] [720p] ][ 9 2.473298 James Bond 11 - MoonRaker [1979] ][ 17 0.660664 The Day After Tomorrow 1080p ][ 18 5.708070 A.Beautiful.Mind.2001.720p.HDTV.x264-S26 ][ 22 8.984420 Batman Dark Knight   1080p ][ 23 5.205005 Fantastic Four, Rise Of The Silver Surfer    1080p ][ 26 0.049906 enemy at the gates    1080p ][ 32 1.315114 Counterfeiters.LIMITED.720p.Bluray.x264-SEPTiC ][ 36 4.588228 Shutter   720p ][ 37 8.631833 Corpse Bride   720p ][ 39 8.842121 Jumanji [720p] ][ 40 4.108923 The Mummy 1-2   1080p ][ 42 8.191014 Terminator Salvation Director's cut  ( 2009)    1080p ][ 47 5.977164 Kung Fu Panda [2008]   720p ][ 52 0.478930 Firewall 720p ][ 64 3.457095 Shinjuku Incident ][ 70 7.754090 Seven Pounds    720p ][ 72 2.201533 Harry Potter Collection    720p ][ 77 5.104489 Lord Of The Ring 3 - The Return of the King (ExtEd) [1080p] ][ 84 6.433499 James Bond 11 - MoonRaker [1979] ][ 86 6.852589 Death Sentence   1080p ][ 91 1.054237 Top.Gun.1986.720p.HDDVD.x264-SEPTiC ][ 93 6.090223 Bad Boy I [1995] [720p] ]
[ 1 9.841776 Bad Boy I [1995] [720p] ][ 2 0.854142 Chicken Run [2000] [720p] ][ 3 2.345400 Ice Age 1 [2002] [720p] ][ 9 2.473298 James Bond 11 - MoonRaker [1979] ][ 17 0.660664 The Day After Tomorrow 1080p ][ 18 5.708070 A.Beautiful.Mind.2001.720p.HDTV.x264-S26 ][ 23 5.205005 Fantastic Four, Rise Of The Silver Surfer    1080p ][ 26 0.049906 enemy at the gates    1080p ][ 27 0.407093 Assassination of a High School President 720p ][ 29 4.144655 Face.Off.1997.720p ][ 33 1.901524 Casablana.1942.1080p.BluRay.x264.Codres ][ 36 4.588228 Shutter   720p ][ 37 8.631833 Corpse Bride   720p ][ 38 0.922589 Juno 2007 720p ][ 39 8.842121 Jumanji [720p] ][ 40 4.108923 The Mummy 1-2   1080p ][ 42 8.191014 Terminator Salvation Director's cut  ( 2009)    1080p ][ 45 9.855380 Black.Mask.Blu-ray.720p.x264.DTS-HDC.AfterFX@SiLUHD ][ 47 5.977164 Kung Fu Panda [2008]   720p ][ 52 0.478930 Firewall 720p ][ 64 3.457095 Shinjuku Incident ][ 70 7.754090 Seven Pounds    720p ][ 71 4.864067 Deep Sea ][ 72 2.201533 Harry Potter Collection    720p ][ 77 5.104489 Lord Of The Ring 3 - The Return of the King (ExtEd) [1080p] ][ 79 7.089044 National Tressure 1 - 1080p ][ 81 9.862417 Equilibrium 720p ][ 91 1.054237 Top.Gun.1986.720p.HDDVD.x264-SEPTiC ][ 93 6.090223 Bad Boy I [1995] [720p] ][ 101 2.422137 Dragon Wars   720p ]
[ 1 9.841776 Bad Boy I [1995] [720p] ][ 2 0.854142 Chicken Run [2000] [720p] ][ 3 2.345400 Ice Age 1 [2002] [720p] ][ 8 8.851587 Pirates of Caribbean 1-2-3 Collection   1080p ][ 9 2.473298 James Bond 11 - MoonRaker [1979] ][ 11 3.092086 American Virgin 720P ][ 17 0.660664 The Day After Tomorrow 1080p ][ 18 5.708070 A.Beautiful.Mind.2001.720p.HDTV.x264-S26 ][ 19 6.684097 James Bond 03 - GoldFinger [1964] ][ 23 5.205005 Fantastic Four, Rise Of The Silver Surfer    1080p ][ 26 0.049906 enemy at the gates    1080p ][ 27 0.407093 Assassination of a High School President 720p ][ 33 1.901524 Casablana.1942.1080p.BluRay.x264.Codres ][ 35 0.955647 Eagle Eye    1080p ][ 36 4.588228 Shutter   720p ][ 38 0.922589 Juno 2007 720p ][ 42 8.191014 Terminator Salvation Director's cut  ( 2009)    1080p ][ 45 9.855380 Black.Mask.Blu-ray.720p.x264.DTS-HDC.AfterFX@SiLUHD ][ 47 5.977164 Kung Fu Panda [2008]   720p ][ 51 4.375161 The Dark Knight [2008]   1080p ][ 52 0.478930 Firewall 720p ][ 64 3.457095 Shinjuku Incident ][ 70 7.754090 Seven Pounds    720p ][ 71 4.864067 Deep Sea ][ 72 2.201533 Harry Potter Collection    720p ][ 76 9.008009 Eurotrip    720p ][ 79 7.089044 National Tressure 1 - 1080p ][ 81 9.862417 Equilibrium 720p ][ 82 2.474496 Transformer 1 & 2 Collection   1080p ][ 83 7.945302 8 Mile [2002] [720p] ][ 88 9.753327 Fifty Dead Men Walking   1080p ][ 89 1.512438 Night Train   720p ][ 91 1.054237 Top.Gun.1986.720p.HDDVD.x264-SEPTiC ][ 93 6.090223 Bad Boy I [1995] [720p] ][ 101 2.422137 Dragon Wars   720p ][ 102 1.526266 10,000 BC 720p ]
[ 2 0.854142 Chicken Run [2000] [720p] ][ 3 2.345400 Ice Age 1 [2002] [720p] ][ 8 8.851587 Pirates of Caribbean 1-2-3 Collection   1080p ][ 16 3.309936 Race To Witch Mountain [1080p] ][ 17 0.660664 The Day After Tomorrow 1080p ][ 18 5.708070 A.Beautiful.Mind.2001.720p.HDTV.x264-S26 ][ 19 6.684097 James Bond 03 - GoldFinger [1964] ][ 23 5.205005 Fantastic Four, Rise Of The Silver Surfer    1080p ][ 26 0.049906 enemy at the gates    1080p ][ 27 0.407093 Assassination of a High School President 720p ][ 33 1.901524 Casablana.1942.1080p.BluRay.x264.Codres ][ 34 0.740428 SAW IV ][ 36 4.588228 Shutter   720p ][ 38 0.922589 Juno 2007 720p ][ 42 8.191014 Terminator Salvation Director's cut  ( 2009)    1080p ][ 45 9.855380 Black.Mask.Blu-ray.720p.x264.DTS-HDC.AfterFX@SiLUHD ][ 47 5.977164 Kung Fu Panda [2008]   720p ][ 51 4.375161 The Dark Knight [2008]   1080p ][ 52 0.478930 Firewall 720p ][ 53 5.482094 Chicken Little 1080p ][ 61 8.070845 Pursuit_Of_Happyness[2006]   720p ][ 64 3.457095 Shinjuku Incident ][ 68 4.854085 Indiana.Jones.and.the.Kingdom.of.the.Crystal.Skull    1080p ][ 69 3.468441 Yes Man   720p ][ 70 7.754090 Seven Pounds    720p ][ 71 4.864067 Deep Sea ][ 72 2.201533 Harry Potter Collection    720p ][ 76 9.008009 Eurotrip    720p ][ 79 7.089044 National Tressure 1 - 1080p ][ 81 9.862417 Equilibrium 720p ][ 82 2.474496 Transformer 1 & 2 Collection   1080p ][ 83 7.945302 8 Mile [2002] [720p] ][ 89 1.512438 Night Train   720p ][ 90 6.126213 X-Men 1 - 1080p ][ 91 1.054237 Top.Gun.1986.720p.HDDVD.x264-SEPTiC ][ 93 6.090223 Bad Boy I [1995] [720p] ][ 101 2.422137 Dragon Wars   720p ][ 102 1.526266 10,000 BC 720p ][ 106 4.813766 Dragon Wars   720p ][ 108 1.580269 Dragonball Evolution [2009] [720p] ][ 119 9.744142 Death Sentence   1080p ][ 123 7.144789 Species - The Awakening   720p ]
[ 2 0.854142 Chicken Run [2000] [720p] ][ 3 2.345400 Ice Age 1 [2002] [720p] ][ 4 6.145100 contents.txt ][ 8 8.851587 Pirates of Caribbean 1-2-3 Collection   1080p ][ 16 3.309936 Race To Witch Mountain [1080p] ][ 17 0.660664 The Day After Tomorrow 1080p ][ 18 5.708070 A.Beautiful.Mind.2001.720p.HDTV.x264-S26 ][ 19 6.684097 James Bond 03 - GoldFinger [1964] ][ 20 3.718962 Blood.Diamond.2006.1080p.HDDVD.x264-DEFiNiTE ][ 23 5.205005 Fantastic Four, Rise Of The Silver Surfer    1080p ][ 25 0.976694 Blood River 720P ][ 26 0.049906 enemy at the gates    1080p ][ 27 0.407093 Assassination of a High School President 720p ][ 34 0.740428 SAW IV ][ 36 4.588228 Shutter   720p ][ 38 0.922589 Juno 2007 720p ][ 42 8.191014 Terminator Salvation Director's cut  ( 2009)    1080p ][ 45 9.855380 Black.Mask.Blu-ray.720p.x264.DTS-HDC.AfterFX@SiLUHD ][ 47 5.977164 Kung Fu Panda [2008]   720p ][ 49 1.560757 Pitch Black [720p] ][ 50 6.134728 Lord Of The Ring  - Return Of The King   720p ][ 51 4.375161 The Dark Knight [2008]   1080p ][ 52 0.478930 Firewall 720p ][ 53 5.482094 Chicken Little 1080p ][ 55 5.938738 X-Men 2   1080p ][ 61 8.070845 Pursuit_Of_Happyness[2006]   720p ][ 69 3.468441 Yes Man   720p ][ 71 4.864067 Deep Sea ][ 72 2.201533 Harry Potter Collection    720p ][ 76 9.008009 Eurotrip    720p ][ 79 7.089044 National Tressure 1 - 1080p ][ 81 9.862417 Equilibrium 720p ][ 82 2.474496 Transformer 1 & 2 Collection   1080p ][ 83 7.945302 8 Mile [2002] [720p] ][ 89 1.512438 Night Train   720p ][ 90 6.126213 X-Men 1 - 1080p ][ 91 1.054237 Top.Gun.1986.720p.HDDVD.x264-SEPTiC ][ 93 6.090223 Bad Boy I [1995] [720p] ][ 96 0.660276 Sky.Captain.And.The.World.Of.Tomorrow[www.x1949x.com]    720p ][ 98 5.818378 Monster Inc [2001] [720p] ][ 101 2.422137 Dragon Wars   720p ][ 102 1.526266 10,000 BC 720p ][ 106 4.813766 Dragon Wars   720p ][ 108 1.580269 Dragonball Evolution [2009] [720p] ][ 119 9.744142 Death Sentence   1080p ][ 123 7.144789 Species - The Awakening   720p ][ 133 2.038412 Finding Nemo [2003] [720p] ][ 135 6.449867 Death Sentence   1080p ][ 140 2.605059 Golden Compass   720p ]
[ 2 0.854142 Chicken Run [2000] [720p] ][ 3 2.345400 Ice Age 1 [2002] [720p] ][ 4 6.145100 contents.txt ][ 8 8.851587 Pirates of Caribbean 1-2-3 Collection   1080p ][ 16 3.309936 Race To Witch Mountain [1080p] ][ 17 0.660664 The Day After Tomorrow 1080p ][ 18 5.708070 A.Beautiful.Mind.2001.720p.HDTV.x264-S26 ][ 19 6.684097 James Bond 03 - GoldFinger [1964] ][ 20 3.718962 Blood.Diamond.2006.1080p.HDDVD.x264-DEFiNiTE ][ 25 0.976694 Blood River 720P ][ 26 0.049906 enemy at the gates    1080p ][ 34 0.740428 SAW IV ][ 36 4.588228 Shutter   720p ][ 38 0.922589 Juno 2007 720p ][ 42 8.191014 Terminator Salvation Director's cut  ( 2009)    1080p ][ 45 9.855380 Black.Mask.Blu-ray.720p.x264.DTS-HDC.AfterFX@SiLUHD ][ 47 5.977164 Kung Fu Panda [2008]   720p ][ 48 4.432253 American Beauty   720p ][ 49 1.560757 Pitch Black [720p] ][ 51 4.375161 The Dark Knight [2008]   1080p ][ 52 0.478930 Firewall 720p ][ 53 5.482094 Chicken Little 1080p ][ 55 5.938738 X-Men 2   1080p ][ 69 3.468441 Yes Man   720p ][ 71 4.864067 Deep Sea ][ 72 2.201533 Harry Potter Collection    720p ][ 76 9.008009 Eurotrip    720p ][ 78 9.835793 Jurassic Park 1-3  1080p ][ 79 7.089044 National Tressure 1 - 1080p ][ 81 9.862417 Equilibrium 720p ][ 82 2.474496 Transformer 1 & 2 Collection   1080p ][ 83 7.945302 8 Mile [2002] [720p] ][ 89 1.512438 Night Train   720p ][ 90 6.126213 X-Men 1 - 1080p ][ 91 1.054237 Top.Gun.1986.720p.HDDVD.x264-SEPTiC ][ 93 6.090223 Bad Boy I [1995] [720p] ][ 95 2.638061 Golden Compass   720p ][ 96 0.660276 Sky.Captain.And.The.World.Of.Tomorrow[www.x1949x.com]    720p ][ 97 4.410551 Shoot.Em.Up.720p.Bluray.x264-SEPTiC ][ 98 5.818378 Monster Inc [2001] [720p] ][ 101 2.422137 Dragon Wars   720p ][ 102 1.526266 10,000 BC 720p ][ 105 9.477671 Around The World In 80 Days [2004] [720p] ][ 106 4.813766 Dragon Wars   720p ][ 108 1.580269 Dragonball Evolution [2009] [720p] ][ 119 9.744142 Death Sentence   1080p ][ 123 7.144789 Species - The Awakening   720p ][ 124 8.297857 Lord of war   720p ][ 125 9.944070 The Extreme Movie 720P ][ 133 2.038412 Finding Nemo [2003] [720p] ][ 135 6.449867 Death Sentence   1080p ][ 138 0.487034 Spiderman 1   1080p ][ 140 2.605059 Golden Compass   720p ][ 143 1.340041 Final Fantasy VII Advent Children [2005] [1080p] ][ 145 4.560221 CJ7   720p ][ 146 5.709145 Night At The Museum 2 [720p] ]
